tests/installation/partitioning_raid.pm

```diff
@@ -148,45 +148,71 @@ sub run() {
 
     # user defined
     send_key $cmd{custompart};
+    assert_screen 'custompart_option-selected';
     send_key $cmd{next};
     assert_screen 'custompart';
 
     send_key "tab";
-    send_key "down";    # select disks
+    assert_screen 'custompart_systemview-selected';
+    send_key "down";
+    assert_screen 'partitioning_raid-hard_disks-selected';
+
     if (get_var("OFW")) {    ## no RAID /boot partition for ppc
+        wait_idle 3;
```

Comment:
As long as Sergio is using any kind of sleep just to get screenshots, save these as needles and then convert the code into assert_screen we don't need to care not I then suggest to use the interactive mode or check_screen instead. Changing the code invalidates previous validation runs though. In the end I can not forbid you to use wait_idle or sleep but I strongly discourage it.   Reasons: no user looks on his stop watch, waiting for a certain amount of seconds before he does the next action. Second, any value selection is an empirical selection that will most likely eventually fail, e.g. when testing on another machine, etc.